framework/auth/campaigns.py

```diff
@@ -0,0 +1,46 @@
+import httplib as http
+
+from framework.exceptions import HTTPError
+
+from website import mails
+
+VALID_CAMPAIGNS = (
+    'prereg',
+)
+
+EMAIL_TEMPLATE_MAP = {
+    'prereg': mails.CONFIRM_EMAIL_PREREG
```

Comment:
Given my suggestion of changing the name of the campaign identifier:

Have you considered merging `EMAIL_TEMPLATE_MAP` with `VALID_CAMPAIGNS` above? Specifying the campaign name key only once might help to keep the code maintainable.

eg 
```
VALID_CAMPAIGNS = {
    'prereg': {
        'email_template': mails.CONFIRM_EMAIL_PREREG,
        'redirect_url': 'http://www.huffingtonpost.com/2015/02/11/australias-oldest-man-sweaters-penguins-oil_n_6660962.html'
    }
}